msgid ""
msgstr ""
"Project-Id-Version: openplotter\n"
"POT-Creation-Date: 2021-01-17 13:13+0100\n"
"PO-Revision-Date: 2021-12-10 12:41\n"
"Last-Translator: \n"
"Language-Team: Polish\n"
"Language: pl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.8.7.1\n"
"X-Poedit-Basepath: ../../..\n"
"Plural-Forms: nplurals=4; plural=(n==1 ? 0 : (n%10>=2 && n%10<=4) && (n%100<12 || n%100>14) ? 1 : n!=1 && (n%10>=0 && n%10<=1) || (n%10>=5 && n%10<=9) || (n%100>=12 && n%100<=14) ? 2 : 3);\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-SearchPath-0: .\n"
"X-Crowdin-Project: openplotter\n"
"X-Crowdin-Project-ID: 289888\n"
"X-Crowdin-Language: pl\n"
"X-Crowdin-File: openplotter-network.po\n"
"X-Crowdin-File-ID: 14\n"

#: networkPostInstall.py:30
msgid "Installing python packages..."
msgstr "Instalowanie pakietów Python..."

#: networkPostInstall.py:33 networkPostInstall.py:51 networkPostInstall.py:63
#: networkPostInstall.py:69 networkPreUninstall.py:31 networkPreUninstall.py:62
msgid "DONE"
msgstr "GOTOWE"

#: networkPostInstall.py:34 networkPostInstall.py:52 networkPostInstall.py:64
#: networkPostInstall.py:70 networkPreUninstall.py:32 networkPreUninstall.py:57
#: networkPreUninstall.py:63
msgid "FAILED: "
msgstr "NIEPOWODZENIE: "

#: networkPostInstall.py:37
msgid "Editing config files..."
msgstr "Edycja plików konfiguracyjnych..."

#: networkPostInstall.py:55
msgid "Enabling services..."
msgstr "Włączanie usług..."

#: networkPostInstall.py:66
msgid "Setting version..."
msgstr "Ustawianie wersji..."

#: networkPreUninstall.py:28
msgid "Removing python packages..."
msgstr "Odinstalowywanie pakietów Python..."

#: networkPreUninstall.py:35
msgid "Restoring network Debian defaults..."
msgstr "Przywracanie domyślnych ustawień sieciowych Debiana..."

#: networkPreUninstall.py:48
msgid "Removing openplotter-network service..."
msgstr "Usuwanie usługi openplotter-network..."

#: networkPreUninstall.py:56
msgid "DONE. PLEASE REBOOT TO RESTORE NETWORK SETTINGS"
msgstr "GOTOWE. PROSZĘ ZRESTARTOWAĆ, ABY PRZYWRÓCIĆ USTAWIENIA SIECIOWE"

#: networkPreUninstall.py:59
msgid "Removing version..."
msgstr "Usuwanie wersji..."

#: openplotterNetwork.py:38
msgid "Network"
msgstr "Sieć"

#: openplotterNetwork.py:48
msgid "Help"
msgstr "Pomoc"

#: openplotterNetwork.py:51
msgid "Settings"
msgstr "Ustawienia"

#: openplotterNetwork.py:54
msgid "Addresses and Ports"
msgstr "Adresy i porty"

#: openplotterNetwork.py:56
msgid "Check Network"
msgstr "Sprawdź stan sieci"

#: openplotterNetwork.py:59
msgid "Install Wifi Drivers"
msgstr "Zainstaluj sterowniki WiFi"

#: openplotterNetwork.py:59
msgid "This does only help for unrecognized usb wlan type:"
msgstr "To może pomóc tylko dla nierozpoznanego typu karty WLAN USB:"

#: openplotterNetwork.py:66
msgid "Access Point"
msgstr "Punkt dostępowy Wifi"

#: openplotterNetwork.py:134
msgid "Network Mode"
msgstr "Tryb sieci"

#: openplotterNetwork.py:139
msgid "AP"
msgstr "AP"

#: openplotterNetwork.py:149
msgid "5 GHz"
msgstr "5 GHz"

#: openplotterNetwork.py:151
msgid "Add Ethernet port to the AP"
msgstr "Dodaj port sieci ethernet (RJ45) do AP"

#: openplotterNetwork.py:174
msgid "Access Point Settings"
msgstr "Ustawienia punktu dostępu (AP)"

#: openplotterNetwork.py:177
msgid "Sharing Internet device"
msgstr "Udostępnianie urządzenia internetowego"

#: openplotterNetwork.py:185
msgid "SSID \n"
"maximum 32 characters"
msgstr "Nazwa sieci (SSID)\n"
"maksymalnie 32 znaki"

#: openplotterNetwork.py:193
msgid "Password \n"
"minimum 8 characters required"
msgstr "Hasło \n"
"wymagane przynajmniej 8 znaków"

#: openplotterNetwork.py:202
msgid "Channel"
msgstr "Kanał WiFi"

#: openplotterNetwork.py:312 openplotterNetwork.py:332
#: openplotterNetwork.py:337 openplotterNetwork.py:387
#: openplotterNetwork.py:391 openplotterNetwork.py:433
#: openplotterNetwork.py:466 openplotterNetwork.py:484
#: openplotterNetwork.py:614
msgid "none"
msgstr "brak"

#: openplotterNetwork.py:404
msgid "Wait please... "
msgstr "Proszę czekać... "

#: openplotterNetwork.py:538
msgid "Your SSID must have a maximum of 32 characters and your password a minimum of 8."
msgstr "Identyfikator SSID może mieć do 32 znaków, a hasło musi liczyć co najmniej 8 znaków."

#: openplotterNetwork.py:589
msgid "OpenPlotter will reboot. If something goes wrong and you are on a headless system, you may not be able to reconnect again.\n\n"
"Are you sure?"
msgstr "OpenPlotter zostanie uruchomiony ponownie. Jeśli coś pójdzie nie tak, a używasz systemu bezmonitorowego (headless), możesz nie być w stanie ponownie się połączyć.\n\n"
"Czy jesteś pewien?"

#: openplotterNetwork.py:590
msgid "Question"
msgstr "Pytanie"

#: openplotterNetwork.py:613 openplotterNetwork.py:614
msgid "wifi access point: "
msgstr "punkt dostępu WiFi: "

#: openplotterNetwork.py:631
msgid "wifi client: "
msgstr "klient WiFi: "

#: openplotterNetwork.py:644
msgid "running"
msgstr "uruchomiony"

#: openplotterNetwork.py:645
msgid "stopped"
msgstr "zatrzymany"

#: openplotterNetwork.py:659
msgid "IP address\n"
msgstr "Adresy IP\n"

#: openplotterNetwork.py:672
msgid "Installing Wifi modules... "
msgstr "Instalowanie modułów WiFi... "

#: openplotterNetwork.py:674
msgid "Done."
msgstr "Gotowe."

#: openplotterNetwork.py:712
msgid "There are conflicts between server connections"
msgstr "Istnieją konflikty pomiędzy połączeniami serwerowymi"

#: openplotterNetwork.py:713
msgid "No conflicts between server connections"
msgstr "Brak konfliktów pomiędzy połączeniami serwerowymi"

#: ports.py:28
msgid "VNC Remote Desktop"
msgstr "Zdalny pulpit VNC"

#: startup.py:46
msgid "Checking WIFI Access Point password..."
msgstr "Sprawdzanie hasła punktu dostępu WIFI..."

#: startup.py:68
msgid "Security warning: You are using the default WIFI Access Point password.\n"
"Please change password in OpenPlotter Network."
msgstr "Niebezpieczeństwo: Używasz domyślnego hasła punktu dostępu WiFi.\n"
"Proszę zmienić hasło sieci OpenPlotter."

#: startup.py:69
msgid "changed"
msgstr "zmieniono"

